socksio/socks5.py

```diff
@@ -97,12 +98,38 @@ def loads(cls, data: bytes) -> "SOCKS5UsernamePasswordReply":
         return cls(success=data == b"\x01\x00")
 
 
-class SOCKS5Request(typing.NamedTuple):
+class SOCKS5CommandRequest(typing.NamedTuple):
     command: SOCKS5Command
     atype: SOCKS5AType
     addr: bytes
     port: int
 
+    @classmethod
+    def from_address_port(
```

Comment:
What are your thoughts on a single parameter, `address` that can either be `str` or `typing.Tuple[str, int]` and renaming to `from_address()` to match Python's APIs using "address" to mean (host, port)?

Reply:
Sounds good to me 👍